sites/www.zooniverse.org.conf

```diff
@@ -83,9 +83,17 @@ server {
         return 301 /projects/arolsen-archives/every-name-counts$1$2$is_args$query_string;
     }
 
+    # ensure the js and CSS assets are served on the same or subdomain
+    # switch back to redirecet once they are served from the static.zooniverse.org domain
+    location ~ \.(js|css)$ {
+        resolver 8.8.8.8;
+        proxy_pass  https://zooniversestatic.z13.web.core.windows.net/$host$request_uri;
+        include /etc/nginx/az-proxy-headers.conf;
+    }
+
     location / {
         # TODO: Long term ensure this eventually points to `static.zooniverse.org` CDN domain for CDN benefits
-        rewrite (?i)\.(jp(e)?g|gif|png|ico|txt|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip|json|js|css)$ https://zooniversestatic.z13.web.core.windows.net/$host$request_uri;
+        rewrite (?i)\.(jp(e)?g|gif|png|ico|txt|mp(3|4)|webm|og(a|g|m|v|x)|spx|opus|pdf|ttf|tar|gz|tgz|bz2|tbz2|zip)$ https://zooniversestatic.z13.web.core.windows.net/$host$request_uri;
```

Comment:
I removed the json assets added for the dev classifier in https://github.com/zooniverse/static/pull/28/files

I can't see these in use at all on our PFE code so I've removed on the www.zooniverse.org domain. We can add this back if / when needed.